### PR TITLE
Debian jessie .deb build config and automated test fixes.

### PR DESCRIPTION
**Pull request by mprzybylski**
Updated deb packaging config to support building packages jessie / jessie-backports

Updated INSTALL.md with Debian build / install instructions

Updated Python and Lua uprobes tests and related code to take advantage of https://github.com/iovisor/bcc/pull/875

Removed `libbcc.so`-related tests from `tests/lua/test_standalone.sh` since one was failing and all were obsoleted by a change to statically link libbcc into `bcc-lua`

### CODE REVIEW

**Comment by drzaeus77** [comment]
[buildbot, ok to test]
[buildbot, add to whitelist]

**Comment by drzaeus77** [comment]
Hi @mprzybylski , the change looks good but I wonder if you can clean up the merge history a little bit (rebase)?

**Comment by mprzybylski** [comment]
@drzaeus77, I'd be happy to.  Bear with me while I RTFM and make that happen...

**Comment by drzaeus77** [comment]
TL;DR: local `git rebase remote/master` followed by `git push origin master -f`

Assuming:
 remote = github.com/iovisor/bcc
and
 origin = github.com/Appdynamics/bcc

I suggest the force push approach since this PR will automatically pick up and test the new changes. Often people try to create a new branch with the rebased changeset, which confuses the PR history.

**Comment by mprzybylski** [comment]
@drzaeus77, this is done per your instructions.

Please let me know if you'd like me to clean up further by squashing commits together.
